spec/controllers/chapters_controller_spec.rb

```diff
@@ -264,6 +262,81 @@
         get :show, params: { work_id: work.id, id: work.chapters.first.id }
       end
     end
+
+    context "when the chapter is the first chapter" do
+      it "increases the hit count" do
+        REDIS_GENERAL.set("work_stats:#{chaptered_work.id}:last_visitor", nil)
+        expect {
+          get :show, params: { work_id: chaptered_work.id, id: first_chapter.id }
+        }.to change { REDIS_GENERAL.get("work_stats:#{chaptered_work.id}:hit_count").to_i }.by(1)
```

Comment:
I'm on the fence... I think instead of looking into specific Redis keys, we should call `StatCounter.hits_to_database` then run expects on the work's `stat_counter.hit_count`, to test the whole process, but that would make the tests slower.